lib/main.js

```diff
@@ -18,43 +18,54 @@ function HTMLUglify(config) {
 
   var salt = config.salt || 'use the force harry';
```

Comment:
@devinus and @elbuo8 I ran a crude speed test. Math.random() is faster.

Results:

```
➜  ~  node speedtest.js
Execution took 894 ms
Execution took 55 ms
```
```
var crypto = require('crypto');

// exec code
var maximumCount = 100000;
var ts = +new Date;
for (i = 0; i < maximumCount; i++) { 
  var random = crypto.pseudoRandomBytes(256);
}

console.log('Execution took ' + ((+new Date) - ts) + ' ms')

var ts = +new Date;
for (i = 0; i < maximumCount; i++) { 
  var random = Math.random() + '';
}

console.log('Execution took ' + ((+new Date) - ts) + ' ms')
```